events/events.go

```diff
@@ -0,0 +1,15 @@
+package events
+
+import "fmt"
+
+var DEBUG = true
```

Comment:
If we're going to default this to true how about logging a line to warn users to turn it off in "prod"?

Reply:
Raised #2 to start adding a configuration to allow us to turn this on and off.